web-server/server.py

```diff
@@ -1,5 +1,30 @@
-# TODO: Import MongoDB
+import os
 
-# TODO: Serve static files from frontend folder
+from flask import Flask, render_template
+from flask_pymongo import PyMongo
 
-# TODO: Add API endpoint for serving weather data from mongodb.
\ No newline at end of file
+from util import dummy_data
+
+app = Flask(__name__)
+
+# Retrieves the variables necessary to assemble the MONGO URI
+MONGODB_USERNAME = os.environ.get('MONGODB_USERNAME')
+MONGODB_PASSWORD = os.environ.get('MONGODB_PASSWORD')
+MONGODB_HOSTNAME = os.environ.get('MONGODB_HOSTNAME')
+MONGODB_DATABASE = os.environ.get('MONGODB_DATABASE')
+
+app.config["MONGO_URI"] = f'mongodb://{MONGODB_USERNAME}:{MONGODB_PASSWORD}' \
+    '@{MONGODB_HOSTNAME}:27017/{MONGODB_DATABASE}'
```

Comment:
Legge til en standard verdi som brukes om ikke env variabel er satt?

Evt. pushe en eksempel environment fil til web-server-mappen
^^.... Eller den vil vel kanskje blir spist opp av docker build greien hmm. Så da må vi isåfall ha dockerignore fil i tillegg